technic/machines/network.lua

```diff
@@ -654,7 +654,7 @@ function technic.network_run(network_id)
 	end
 	--dprint("Total RE demand:"..RE_eu_demand)
 
-	technic.network_infotext(network_id, S("@1. Supply: @2 Demand: @3",
+	technic.network_infotext(network_id, S("@1. Supply: @2 Demand: @3",
```

Comment:
Answering to https://github.com/mt-mods/technic/pull/255/files#r766267806
Yes here it is also intentional to use non breaking space, I think not as important but better than without.
Space positions are correct to keep "Supply:" together with its amount and similar for "Demand:" to display it correctly with all screen size and font variations.
Normal space between "Supply: @2" and "Demand: @3" is also correct because that's part where possible line breaks  should happen, same for first normal space.

Reply:
non-breaking spaces restored here https://github.com/mt-mods/technic/pull/255/commits/dbb7798f43b669f93e4b968dff2bd011ac27512c